gobblin-utility/src/main/java/org/apache/gobblin/util/JobLauncherUtils.java

```diff
@@ -123,6 +126,20 @@ public static List<WorkUnit> flattenWorkUnits(Collection<WorkUnit> workUnits) {
     return flattenedWorkUnits;
   }
 
+  /** @return flattened list of {@link WorkUnit}s loaded from `path`, which may possibly hold a multi-work unit */
+  public static List<WorkUnit> loadFlattenedWorkUnits(FileSystem fs, Path path) throws IOException {
+    WorkUnit workUnit = path.getName().endsWith(MULTI_WORK_UNIT_FILE_EXTENSION)
+        ? MultiWorkUnit.createEmpty()
+        : WorkUnit.createEmpty();
+    SerializationUtils.deserializeState(fs, path, workUnit);
+
+    if (workUnit instanceof MultiWorkUnit) {
+      return JobLauncherUtils.flattenWorkUnits(((MultiWorkUnit) workUnit).getWorkUnits());
+    } else {
```

Comment:
Small nit, we don't need else return after a return statement

Reply:
I agree it's not necessary, but I prefer the parallel indentation of `if`/`else`, which I find clearer.  TBH, I almost went w/ the ternary operator... but it seemed harder to read.

Reply:
I'd prefer the current over ternary, I guess because there's quite a bit of casting involved here so it's not as easy to follow what's going on.